tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/GitHubLabelsTool/GitHubLabelsTool.cs

```diff
@@ -0,0 +1,251 @@
+
+
+namespace Azure.Sdk.Tools.Cli.Tools
+{
+
+    [McpServerToolType, Description("Tools for working with GitHub service labels from the Azure SDK Tools common-labels.csv")]
+    public class GitHubLabelsTool(
+        ILogger<GitHubLabelsTool> logger,
+        IOutputService output,
+        ILabelHelper labelHelper,
+        IGitHubService githubService
+    ) : MCPTool
+    {
+        private const string serviceLabelColorCode = "e99695"; // color code for service labels in common-labels.csv
+
+        //command names
+        private const string checkServiceLabelCommandName = "check-service-label";
+        private const string createServiceLabelCommandName = "create-service-label";
+
+        // Command options
+        private readonly Option<string> serviceLabelOpt = new(["--service", "-s"], "Proposed Service name used to create a PR for a new label.") { IsRequired = true };
+        private readonly Option<string> documentationLinkOpt = new(["--link", "-l"], "Brand documentation link used to create a PR for a new label.") { IsRequired = true };
+
+        public override Command GetCommand()
+        {
+            var command = new Command("github-labels", "GitHub service labels tools");
+            var subCommands = new[]
+            {
+                new Command(checkServiceLabelCommandName, "Check if a service label exists in the common labels CSV") { serviceLabelOpt },
+                new Command(createServiceLabelCommandName, "Creates a PR for a new label given a proposed label and brand documentation.") { serviceLabelOpt, documentationLinkOpt },
+            };
+
+            foreach (var subCommand in subCommands)
+            {
+                subCommand.SetHandler(async ctx => { await HandleCommand(ctx, ctx.GetCancellationToken()); });
+                command.AddCommand(subCommand);
+            }
+            return command;
+        }
+
+        public override async Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            var command = ctx.ParseResult.CommandResult.Command.Name;
+            var commandParser = ctx.ParseResult;
+
+            switch (command)
+            {
+                case checkServiceLabelCommandName:
+                    var serviceLabel = commandParser.GetValueForOption(serviceLabelOpt);
+                    var result = await CheckServiceLabel(serviceLabel);
+                    ctx.ExitCode = ExitCode;
+                    output.Output(result);
+                    return;
+                case createServiceLabelCommandName:
+                    var proposedServiceLabel = commandParser.GetValueForOption(serviceLabelOpt);
+                    var documentationLink = commandParser.GetValueForOption(documentationLinkOpt);
+                    var createdPRResult = await CreateServiceLabel(proposedServiceLabel, documentationLink ?? "");
+                    ctx.ExitCode = ExitCode;
+                    output.Output(createdPRResult.ToString());
```

Comment:
```suggestion
                    output.Output(createdPRResult);
```

Output handles object types by default and calls the `ToString()` implementation under the hood if the global output format is string. In MCP mode or with `-o json` we output the object as json instead, which will break if the stringified version is passed in here.